src/drivers/mpu6500/mpu6500.cpp

```diff
@@ -753,7 +753,7 @@ int MPU6500::reset()
 	_gyro_range_scale = (0.0174532 / 16.4);//1.0f / (32768.0f * (2000.0f / 180.0f) * M_PI_F);
 	_gyro_range_rad_s = (2000.0f / 180.0f) * M_PI_F;
 
-	set_accel_range(8);
```

Comment:
Yes I think it's ok. In fact we should do it for all drivers (I'll look into that). For the mpu6500 there's an unused define `MPU6500_ACCEL_DEFAULT_RANGE_G` which should be used and changed accordingly.